dfdaemon/config/config.go

```diff
@@ -88,6 +80,82 @@ type Properties struct {
 	// by dfdaemon. Dfdaemon will be able to proxy requests from them with dfget
 	// if the url matches the proxy rules. The first matched rule will be used.
 	HijackHTTPS *HijackConfig `yaml:"hijack_https"`
+
+	// https options
+	Port    uint   `yaml:"port"`
+	HostIP  string `yaml:"hostIp"`
+	CertPem string `yaml:"certpem"`
+	KeyPem  string `yaml:"keypem"`
+
+	// dfget config
+	SuperNodes []string `yaml:"supernodes"`
```

Comment:
Maybe it's better to use "node" here which is the same as the cmd flag `rf.StringSlice("node", nil, "specify the addresses(IP:port) of supernodes that will be passed to dfget.")
`. WDYT?

Reply:
This is surely a bug, and the value of node cli flag will not be used. But I think `supernodes` is clearer for config file. Adding an alias in viper from `supernodes` to `node` should fix that.

Reply:
I added the alias and it now works. My opinion is to keep the `supernodes` field for config file. It's more precise and it's plural, which is more appropriate for a slice.